### ISSUE TITLE
NO_COLOR=1 does not suppress all color escapes

### ISSUE THREAD

**Issue by ustun** [cypress-io/cypress#5513]
### Current behavior:

As mentioned here https://github.com/cypress-io/cypress/issues/1748#issuecomment-390682159 there is a `NO_COLOR=1` flag. (It doesn't seem documented though, I found it via Google. Maybe that's another issue though.)

When this is set, most of the colors are suppressed, but I still see some things in the CI logs that seem like color escape codes.

```
2019-10-29T10:49:45.5683027Z [2K[0G  [32m  ✓[0m[90m should allow brand creation[0m[31m (11492ms)[0m
2019-10-29T10:49:51.2829146Z [2K[0G  [32m  ✓[0m[90m should not allow creating the same brand[0m[31m (5803ms)[0m
2019-10-29T10:49:51.3044692Z 
2019-10-29T10:49:51.3056465Z 
2019-10-29T10:49:51.3057924Z [92m [0m[32m 2 passing[0m[90m (17s)[0m`
```

<img width="892" alt="Screen Shot 2019-10-29 at 2 58 49 PM" src="https://user-images.githubusercontent.com/94374/67765433-30d9e680-fa5d-11e9-9cf0-e23167d270ad.png">

### Desired behavior:

There should be no escape codes in the output, just ascii chars.

### Steps to reproduce: (app code and test code)

1. Set up a CI build on Azure Devops.
2. Run Cypress with `NO_COLOR=1` env var via Docker with the following command:
```
docker run --rm -t -e NO_COLOR=1 -v /work/agent-0/r1/a/src/cypress-files:/src cypress/included:3.4.1 -P /src

```

### Versions

Cypress 3.4.1 running via Docker:
```
docker run --rm -t -e NO_COLOR=1 -v /work/agent-0/r1/a/src/cypress-files:/src cypress/included:3.4.1 -P /src
```

**Comment by jennifer-shehane** [comment]
Yes, this should be the case - we look for this flag in our code here:

https://github.com/cypress-io/cypress/blob/develop/cli/lib/util.js#L224

This flag is then used to set env vars `FORCE_COLOR`, `DEBUG_COLORS` and `MOCHA_COLORS` to `1`. 

There is a note in the changelog when this was initially added: https://docs.cypress.io/guides/references/changelog.html#3-0-0

>We are now aggressively turning on colors when running in CI. If you’re using Jenkins and do not have an ANSI color plugin installed, you can turn off colors with the NO_COLOR=1 environment variable.

I'll have to check more with the team about this - why it's not being turned off specifically when listing our each test result.

**Comment by ustun** [comment]
Thank you @jennifer-shehane for looking into it

**Comment by jimpriest** [comment]
I see this in Jenkins as well. Even with NO_COLOR=1 it makes console output very difficult to read.

**Comment by punkrockbuddyholly** [comment]
Not sure if this will help anyone but I'm running a Cypress docker image in a jenkinsfile and I found that I had to pass the `NO_COLOR=1` env var to the `docker run` part rather than as an env flag passed to `cypress run`. Previously I was running `cypress run --env NO_COLOR=1` and it was making no difference.

**Comment by alexhvastovich** [comment]
Running with Jenkins only and
 ./node_modules/.bin/cypress run --env NO_COLOR=1 
 is not helping

**Comment by pitgrap** [comment]
Another Information to this topic:

We have problems with our Cypress runs stdout in Jenkins. So we added "NO_COLOR=1", but the problem didn't disappear.
I just retestet it locally on my machine and found colors in the summary. 

tl;dr
All outputs are free of color except the summary.  See screenshot
![Screenshot 2022-04-01 133712](https://user-images.githubusercontent.com/227391/161256040-51cf489c-de8e-4bf7-a621-9a4ed0031cc4.png)
Is this a known issue? Shall I open a new issue?

**Comment by atl-mk** [comment]
Hey, this is still an annoying issue

**Comment by atl-mk** [comment]
The issue still occurs
